scipy/stats/_continuous_distns.py

```diff
@@ -7403,7 +7402,8 @@ def get_shape(data, loc, scale):
             # The first-order necessary condition on `shape` can be solved in
             # closed form. It can be used no matter the assumption of the
             # value of the shape.
-            return -len(data) / np.sum(np.log((data - loc)/scale))
+            N = len(data)
+            return - N / (np.sum(np.log(data - loc)) - N*np.log(scale))
```

Comment:
This avoids an underflow in `(data - loc)/scale)`. Please check my math.

Reply:
```
-len(data) / np.sum(np.log((data - loc)/scale))
-N / np.sum(np.log((data - loc)/scale))
-N / np.sum(np.log(data - loc) - np.log(scale))
-N / (np.sum(np.log(data - loc)) - N * np.log(scale))
```

LGTM, the sum is on N and we can split the sum part in 2.